firefox-ios/Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -1,516 +0,0 @@
-        trackingProtectionButton.largeContentTitle = .TabLocationLockButtonLargeContentTitle
-        trackingProtectionButton.accessibilityLabel = .TabLocationLockButtonAccessibilityLabel
-    }
-
-    private(set) lazy var readerModeButton: ReaderModeButton = .build { readerModeButton in
-        readerModeButton.addTarget(self, action: #selector(self.tapReaderModeButton), for: .touchUpInside)
-        readerModeButton.addGestureRecognizer(
-            UILongPressGestureRecognizer(target: self,
-                                         action: #selector(self.longPressReaderModeButton)))
-        readerModeButton.isAccessibilityElement = true
-        readerModeButton.isHidden = true
-        readerModeButton.accessibilityLabel = .TabLocationReaderModeAccessibilityLabel
-        readerModeButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.readerModeButton
-        readerModeButton.accessibilityCustomActions = [
-            UIAccessibilityCustomAction(
-                name: .TabLocationReaderModeAddToReadingListAccessibilityLabel,
-                target: self,
-                selector: #selector(self.readerModeCustomAction))]
-        readerModeButton.showsLargeContentViewer = true
-        readerModeButton.largeContentTitle = .TabLocationReaderModeAccessibilityLabel
-        readerModeButton.largeContentImage = .templateImageNamed(StandardImageIdentifiers.Large.readerView)
-    }
-
-    lazy var reloadButton: StatefulButton = {
-        let reloadButton = StatefulButton(frame: .zero, state: .disabled)
-        reloadButton.addTarget(self, action: #selector(tapReloadButton), for: .touchUpInside)
-        reloadButton.addGestureRecognizer(
-            UILongPressGestureRecognizer(target: self, action: #selector(longPressReloadButton)))
-        reloadButton.imageView?.contentMode = .scaleAspectFit
-        reloadButton.contentHorizontalAlignment = .center
-        reloadButton.accessibilityLabel = .TabLocationReloadAccessibilityLabel
-        reloadButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.reloadButton
-        reloadButton.accessibilityHint = .TabLocationReloadAccessibilityHint
-        reloadButton.isAccessibilityElement = true
-        reloadButton.translatesAutoresizingMaskIntoConstraints = false
-        reloadButton.showsLargeContentViewer = true
-        reloadButton.largeContentTitle = .TabLocationReloadAccessibilityLabel
-        return reloadButton
-    }()
-
-    init(windowUUID: WindowUUID, logger: Logger = DefaultLogger.shared) {
-        self.windowUUID = windowUUID
-        self.logger = logger
-        super.init(frame: .zero)
-        startObservingNotifications(
-            withNotificationCenter: notificationCenter,
-            forObserver: self,
-            observing: [
-                .TrackingProtectionViewControllerDidDismiss,
-                .TrackingProtectionViewControllerDidAppear
```

Comment:
I think we post those under TrackingProtectionViewController, and they can now be removed if nothing is listening to them